installer/windows/Product.wxs

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:fire="http://schemas.microsoft.com/wix/FirewallExtension">
-  <Product Id="{E97D368F-CB18-45B5-8799-1EBB10728A99}" Name="Storj V3 Storage Node" Language="1033" Version="0.0.0" Manufacturer="Storj Labs Inc." UpgradeCode="{B32CF5FC-0665-4712-B88A-22D299565F9E}">
+  <Product Id="{E97D368F-CB18-45B5-8799-1EBB10728A99}" Name="Storj V3 Storage Node" Language="1033" Version="0.22.0.0" Manufacturer="Storj Labs Inc." UpgradeCode="{B32CF5FC-0665-4712-B88A-22D299565F9E}">
```

Comment:
The Version was intentionally set to `0.0.0`. For two reasons:

1. It quickly becomes outdated, as the autoupdater updates the storagenode binary.
2. The Jenkins build does not set the correct value yet. We don't want to update this value manually on every release.

Maybe in future we find a way how to update the version properly on all use cases, but for now we keep it as `0.0.0`.